

diff --git a/DEPS b/DEPS
index 36a160bbacfd..e15c86e8d8a4 100644
--- a/DEPS
+++ b/DEPS
@@ -172,7 +172,7 @@ vars = {
   #
   # Note this revision should be updated with
   # third_party/boringssl/roll_boringssl.py, not roll-dep.
-  'boringssl_revision': '44544d9d2d624cbfff9b1e77cb77f8dfc70d073c',
+  'boringssl_revision': '24e5886c0edfc409c8083d10f9f1120111efd6f5',
   # Three lines of non-changing comments so that
   # the commit queue can handle CLs rolling google-toolbox-for-mac
   # and whatever else without interference from each other.

